libr/util/file.c

```diff
@@ -211,11 +211,18 @@ R_API char *r_file_abspath(const char *file) {
 			return strdup (file);
 		}
 		if (!strchr (file, ':')) {
-			char *abspath = malloc (MAX_PATH);
+			PTCHAR abspath = malloc (MAX_PATH * sizeof (TCHAR));
 			if (abspath) {
-				GetFullPathName (file, MAX_PATH, abspath, NULL);
-				ret = strdup (abspath);
+				PTCHAR f = r_sys_conv_utf8_to_utf16 (file);
+				int s = GetFullPathName (f, MAX_PATH, abspath, NULL);
+				if (s > MAX_PATH) {
+					R_LOG_ERROR ("r_file_abspath/GetFullPathName: Path to file too long.\n");
+				} else if (!s) {
+					r_sys_perror ("r_file_abspath/GetFullPathName");
+				}
+				ret = r_sys_conv_utf16_to_utf8 (abspath);
```

Comment:
Currently radare2 can be built with UNICODE and without, please add corresponding checks, if it is defined then do conv